cypress/fixtures/active_order.json

```diff
@@ -1,48 +1,53 @@
 {
-  "message": "This item was added to your order",
+  "body": {"message": "This item was added to your order",
```

Comment:
Why did you add body?

Reply:
because we had to pass time stamps when order was updated and created and they have to be outside the block with message and items

Reply:
thank you!